bin/pycbc_upload_xml_to_gracedb

```diff
@@ -37,6 +37,26 @@ from glue.ligolw import utils as ligolw_utils
 from ligo.segments import segment, segmentlist
 from pycbc.io.live import gracedb_tag_with_version
 
+def check_gracedb_for_event(gdb_handle, query, far):
+    """
+    Check if there is an event in gracedb with the queried string
+    which matches the FAR given
+    """
+    gdb_events_match_query = list(gdb_handle.events(query=query))
+    ifar = 1. / lal.YRJUL_SI / far
+    for gdb_event in gdb_events_match_query:
+            # Test each gracedb event to see if the FAR matches this event
+            if gdb_event['far'] == far:
```

Comment:
Are FARs not rounded or truncated by GraceDB? I think some tolerance is needed here.

Reply:
good point - my use case was only to avoid events being uploaded more than once. I'll put in a small difference

Reply:
This sounds like something you want to test quite a bit on gracedb-playground.

Reply:
I did a fair bit of testing but removing the actual upload and replacing with a print statement instead of actual upload and this did work in the == case.

I've added a tolerance of 1e-16 (approx 1e-7 in IFAR) to be safe and will retest.